package.json

```diff
@@ -1,34 +1,55 @@
 {
```

Comment:
The PR title and description don't say what the motivation for these changes is
(and "zero-dependency" misleads me to think that some dependency was removed, but it wasn't)